extras/rya.console/src/main/java/mvm/rya/shell/command/mongo/administrative/MongoInstall.java

```diff
@@ -146,7 +146,7 @@ private RyaDetails initializeRyaDetails(final String instanceName, final Install
             .setGeoIndexDetails(new GeoIndexDetails(installConfig.isGeoIndexEnabled()))
             .setTemporalIndexDetails(new TemporalIndexDetails(installConfig.isTemporalIndexEnabled()))
             .setFreeTextDetails(new FreeTextIndexDetails(installConfig.isFreeTextIndexEnabled()))
-            .setEntityCentricIndexDetails(new EntityCentricIndexDetails(installConfig.isEntityCentrixIndexEnabled()))
+            .setEntityCentricIndexDetails(new EntityCentricIndexDetails(false))//not supported in mongo
```

Comment:
Did you update the shell's Install prompt so that it won't even offer EntityCentric and PCJ indices as options?